From 6faa4a421a4d20384d6a205caa80ce8d8f03daa1 Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Wed, 16 Mar 2016 18:37:47 +0200
Subject: [PATCH 0960/1240] fix: config: remove unused config from U-Boot
 (MVEBU_ROM_SILENCE_FLAG_SUPPORT)

- After moving the binary extension to ATF no need for silent flag

Change-Id: I4a75cb7574a5545e6a505bddb18b27db352e0068
Signed-off-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/28312
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Haim Boot <hayim@marvell.com>
---
 arch/arm/cpu/mvebu-common/Kconfig      | 9 ---------
 configs/mvebu_apn806_defconfig         | 1 -
 configs/mvebu_apn806_dop_defconfig     | 1 -
 configs/mvebu_apn806_pd_defconfig      | 1 -
 configs/mvebu_armada70x0_defconfig     | 1 -
 configs/mvebu_armada70x0_dop_defconfig | 1 -
 configs/mvebu_armada70x0_pd_defconfig  | 1 -
 configs/mvebu_armada70x0_rz_defconfig  | 1 -
 8 files changed, 16 deletions(-)

diff --git a/arch/arm/cpu/mvebu-common/Kconfig b/arch/arm/cpu/mvebu-common/Kconfig
index aaef785..4e643d2 100644
--- a/arch/arm/cpu/mvebu-common/Kconfig
+++ b/arch/arm/cpu/mvebu-common/Kconfig
@@ -120,12 +120,3 @@ config MVEBU_SYS_INFO
 	  If you want to transfer information
 	  from spl to u-boot, you need to choose
 	  this option.
-
-config MVEBU_ROM_SILENCE_FLAG_SUPPORT
-	bool "BootROM silence flag support"
-	default n
-	help
-	  Choose this option to add support for
-	  silence parameter passed to SPL by
-	  BootROM for suppressing serial all
-	  port output during SPL run.
diff --git a/configs/mvebu_apn806_defconfig b/configs/mvebu_apn806_defconfig
index 9342ac9..6b01311 100644
--- a/configs/mvebu_apn806_defconfig
+++ b/configs/mvebu_apn806_defconfig
@@ -35,4 +35,3 @@ CONFIG_MVEBU_COMPHY_SUPPORT=y
 CONFIG_MVEBU_THERMAL_SENSOR=y
 CONFIG_MVEBU_THERMAL_SENSOR_28NM_V2=y
 CONFIG_DEVEL_BOARD=y
-CONFIG_MVEBU_ROM_SILENCE_FLAG_SUPPORT=y
diff --git a/configs/mvebu_apn806_dop_defconfig b/configs/mvebu_apn806_dop_defconfig
index efcbf64..5831da6 100644
--- a/configs/mvebu_apn806_dop_defconfig
+++ b/configs/mvebu_apn806_dop_defconfig
@@ -5,7 +5,6 @@ CONFIG_SYS_EXTRA_OPTIONS="ARM64"
 CONFIG_MVEBU_MPP_BUS=y
 +S:CONFIG_MVEBU_SPL_DDR_OVER_PCI_SUPPORT=y
 +S:CONFIG_MVEBU_SPL_DDR_OVER_PCI_SWITCH=y
-+S:CONFIG_MVEBU_ROM_SILENCE_FLAG_SUPPORT=y
 +S:CONFIG_MVEBU_SPL_SAR_DUMP=y
 +S:CONFIG_DEVEL_BOARD=y
 +S:# CONFIG_MULTI_DT_FILE is not set
diff --git a/configs/mvebu_apn806_pd_defconfig b/configs/mvebu_apn806_pd_defconfig
index 22ab8ac..115bac7 100644
--- a/configs/mvebu_apn806_pd_defconfig
+++ b/configs/mvebu_apn806_pd_defconfig
@@ -6,7 +6,6 @@ CONFIG_SYS_EXTRA_OPTIONS="ARM64"
 CONFIG_MVEBU_CCU=y
 CONFIG_MVEBU_RFU=y
 CONFIG_MVEBU_MPP_BUS=y
-+S:CONFIG_MVEBU_ROM_SILENCE_FLAG_SUPPORT=y
 +S:CONFIG_DEVEL_BOARD=y
 +S:# CONFIG_MULTI_DT_FILE is not set
 CONFIG_CMD_BOOTD=y
diff --git a/configs/mvebu_armada70x0_defconfig b/configs/mvebu_armada70x0_defconfig
index 1d63ad4..2a69c5e 100644
--- a/configs/mvebu_armada70x0_defconfig
+++ b/configs/mvebu_armada70x0_defconfig
@@ -49,4 +49,3 @@ CONFIG_DEVEL_BOARD=y
 # CONFIG_MULTI_DT_FILE is not set
 # CONFIG_MVEBU_LLC_ENABLE is not set
 CONFIG_ENV_IS_IN_BOOTDEV=y
-CONFIG_MVEBU_ROM_SILENCE_FLAG_SUPPORT=y
diff --git a/configs/mvebu_armada70x0_dop_defconfig b/configs/mvebu_armada70x0_dop_defconfig
index 622b5ec..bbeb1f1 100644
--- a/configs/mvebu_armada70x0_dop_defconfig
+++ b/configs/mvebu_armada70x0_dop_defconfig
@@ -5,7 +5,6 @@ CONFIG_SYS_EXTRA_OPTIONS="ARM64"
 +S:CONFIG_AP806_Z_SUPPORT=y
 CONFIG_MVEBU_MPP_BUS=y
 +S:CONFIG_MVEBU_SPL_DDR_OVER_PCI_SUPPORT=y
-+S:CONFIG_MVEBU_ROM_SILENCE_FLAG_SUPPORT=y
 +S:CONFIG_MVEBU_SPL_SAR_DUMP=y
 +S:CONFIG_DEVEL_BOARD=y
 +S:# CONFIG_MULTI_DT_FILE is not set
diff --git a/configs/mvebu_armada70x0_pd_defconfig b/configs/mvebu_armada70x0_pd_defconfig
index 793cdf7..5346d7c 100644
--- a/configs/mvebu_armada70x0_pd_defconfig
+++ b/configs/mvebu_armada70x0_pd_defconfig
@@ -9,7 +9,6 @@ CONFIG_MVEBU_IOB=y
 CONFIG_MVEBU_RFU=y
 CONFIG_MVEBU_MPP_BUS=y
 # CONFIG_MVEBU_TOOLS_SUPPORT is not set
-+S:CONFIG_MVEBU_ROM_SILENCE_FLAG_SUPPORT=y
 +S:CONFIG_DEVEL_BOARD=y
 # CONFIG_MVEBU_SAR is not set
 +S:# CONFIG_MULTI_DT_FILE is not set
diff --git a/configs/mvebu_armada70x0_rz_defconfig b/configs/mvebu_armada70x0_rz_defconfig
index 3b16269..9f95aef 100644
--- a/configs/mvebu_armada70x0_rz_defconfig
+++ b/configs/mvebu_armada70x0_rz_defconfig
@@ -50,4 +50,3 @@ CONFIG_DEVEL_BOARD=y
 # CONFIG_MVEBU_LLC_ENABLE is not set
 CONFIG_AP806_Z_SUPPORT=y
 CONFIG_ENV_IS_IN_BOOTDEV=y
-CONFIG_MVEBU_ROM_SILENCE_FLAG_SUPPORT=y
-- 
1.9.1

